raisinbread/RB_files/RB_Config.sql

```diff
@@ -33,8 +33,8 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (33,37,'/PATH/TO/Genomic
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (34,38,'/data/uploads/');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (35,40,'main.css');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (36,41,'25');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (37,44,'localhost');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (38,45,'http://localhost');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (37,44,'https://jsaigle-dev.loris.ca');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (38,45,'https://jsaigle-dev.loris.ca');
```

Comment:
This isn't right

Reply:
Hmmm I think this may be a side-effect of the RB exporter script. Maybe worth opening a ticket for it

Reply:
@driusan I reverted these changes.